src/lib/y2network/autoinst_profile/networking_section.rb

```diff
@@ -113,6 +117,7 @@ def self.new_from_network(config)
         return result unless config
 
         result.managed = config.backend?(:network_manager)
+        result.backend = config.backend&.id.to_s if config.backend
```

Comment:
We are using a string instead of a symbol, which is OK. But, please, add a test to make sure we are exporting the right value.